### PR TITLE
fix: Issue #5286: Add some words that start with "un" to the US dictionary

### PR DESCRIPTION
**Pull request by klonos**
Fixes https://github.com/streetsidesoftware/cspell/issues/5286

# Add/Fix Dictionary

Dictionary: US

## Description

CSpell has been nagging about these words found in the Backdrop CMS codebase, but they are not specific to that.

It also adds the word `instantiatable`, since it is an actual word (see https://en.wiktionary.org/wiki/instantiatable), as well as its `uninstantiatable` counterpart.

## References

https://github.com/backdrop/backdrop-issues/issues/6302

## Checklist

- [x] By submitting this pull-request, you agree to follow our [Code of Conduct](https://github.com/streetsidesoftware/cspell-dicts/blob/main/CODE_OF_CONDUCT.md)
- [x] Verify that the title starts with the correct prefix:
  - `fix:` - for minor changes like adding words or fixing spelling issues.
  - `feat:` - for a significant change like adding a whole new set of words to a dictionary.
  - `feat!:` - for breaking changes, like file format or licensing changes.
  - `chore:` - for changes that do not impact the content of dictionaries.

### CODE REVIEW

**Comment by Jason3S** [comment]
@klonos,

Thank you for taking a look at this.

Some of the terms are more programming related than part of general English.

For example, what does `unescapes` mean when it is not in a programming context?

The other drawback to adding them to the US English dictionary is that they are only available to those who use `en-US` and not `en-GB`.

If a term is primarily used when writing code, please add it to [dictionaries/software-terms/src/software-terms.txt](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/src/software-terms.txt).

**Comment by Jason3S** [comment]
@klonos,

Will you be able to adjust the PR as suggested?

**Comment by klonos** [comment]
@Jason3S sorry for being unresponsive here, and yes, I plan to loop back to this and adjust as per your suggestions. We have been busy in the Backdrop CMS project, with the feature freeze being on May 1st, and now the upcoming release in May 15. I'm hoping to slate some time for this on the weekend of May 18-19. I would appreciate it if you kept this open if not too much trouble please.

**Comment by Jason3S** [comment]
@klonos,

No a problem. I'll keep the PR open. I appreciate the help.

**Comment by Jason3S** [comment]
@klonos,
Would you like some help with this PR?

File: dictionaries/en_US/src/additional_words.txt [hunk 1]
```diff
@@ -54,12 +55,33 @@ rebrand
 rebranding
 teleconverter
 tricep
+unaggregated
+uncustomized
```
Comment (ssbarnea) [inline dictionaries/en_US/src/additional_words.txt:59]: `uncontained` is missing, found it today!

**Pull request closed without merging**